src/server/v2.0/handler/preheat.go

```diff
@@ -350,7 +359,7 @@ func (api *preheatAPI) PingInstances(ctx context.Context, params operation.PingI
 			return operation.NewPingInstancesNotFound()
 		}
 		if err != nil {
-			api.SendError(ctx, err)
+			return api.SendError(ctx, err)
```

Comment:
have you check other API handlers to make sure there are not same bugs existing？

Reply:
yup